ndt7.go

```diff
@@ -110,20 +131,32 @@ func (c *Client) discoverServer(ctx context.Context) (string, error) {
 }
 
 // doConnect establishes a websocket connection.
-func (c *Client) doConnect(ctx context.Context, URLPath string) (*websocket.Conn, error) {
+func (c *Client) doConnect(
```

Comment:
This new function declaration syntax: Why?

Reply:
I wanted to break long lines and invoking `go fmt ./...` did not cause any change in the code base, so I assumed this formatting was really okay.